### PR TITLE
move sample test suite to test/sample

### PR DESCRIPTION
**Pull request by lionelvillard**

### CODE REVIEW

**Comment by lionelvillard** [comment]
ping. @slinkydeveloper

File: test/sample/suite_test.go [hunk 1]
```diff
@@ -41,7 +41,6 @@ func TestMain(m *testing.M) {
 		NewSuite(m).
 		Configure(&myconfig).
 		Require(sequencestepper.Component).
-		Require(github.Component).
```
Comment (slinkydeveloper) [inline test/sample/suite_test.go:44]: Why you don't require github source anymore?

Comment (lionelvillard) [reply test/sample/suite_test.go:44]: for no good reason. I'll add it later on.

**Pull request merged**